Purchases/Misc/RCSystemInfo.m

```diff
@@ -146,6 +146,10 @@ - (BOOL)isAppExtension {
 
 #endif
 
+- (BOOL)isOperatingSystemAtLeastVersion:(NSOperatingSystemVersion)version {
+    return [NSProcessInfo.processInfo isOperatingSystemAtLeastVersion:version];
```

Comment:
RCReceiptFetcher could be calling this instead, but maybe it'd be better to do it as a separate thing, since again, this PR got much bigger than I anticipated.